addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -427,7 +427,9 @@ export const editorCommands = {
         editor.document.execCommand('removeFormat');
         for (const node of getTraversedNodes(editor.editable)) {
             // The only possible background image on text is the gradient.
-            closestElement(node).style.backgroundImage = '';
+            const closest = closestElement(node);
+            closest.removeAttribute("style");
+            closest.removeAttribute("color");
```

Comment:
@maad-odoo no need to do it separately instead:
` closest.removeAttribute("style color");` 
can you add test-cases relevant to it.